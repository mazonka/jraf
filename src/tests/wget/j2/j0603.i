command=jw md 0 /aaa/bbb : get 0 / : save 0 /aaa/c1 10 MTIzNDU2Nzg5MA== : get 0 / : put 0 /aaa/c2 0 10 MTIzNDU2Nzg5MA== : get 0 / : put 0 /aaa/c2 10 10 MTIzNDU2Nzg5MA== : get 0 / : read 0 /aaa : get 0 / : rm 0 /aaa/c1 : get 0 / : mv 0 /aaa/c2 /aaa/c3 : get 0 / : read 0 /aaa : get 0 /