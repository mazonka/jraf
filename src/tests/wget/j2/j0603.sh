#!/bin/sh

cwd=`sh execdir.sh`
wr=$cwd/wroot
echo ""
echo "=="
ls -1 $wr/aaa
rm -rf $wr



