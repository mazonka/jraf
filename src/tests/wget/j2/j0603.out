OK /aaa/bbb : OK 1 -1 : OK 10 : OK 2 -1 : OK 10 : OK 3 -1 : OK 20 : OK 4 -1 : OK 4 -1 3 1 -1 bbb 1 10 c1 2 20 c2 : OK 4 -1 : OK /aaa/c1 : OK 5 -1 : OK /aaa/c3 : OK 7 -1 : OK 7 -1 2 1 -1 bbb 1 20 c3 : OK 7 -1
==
bbb
c3
